src/data/dem_data.js

```diff
@@ -57,22 +57,14 @@ class DEMData {
         this.loaded = !!data;
     }
 
-    loadFromImage(data: RGBAImage) {
+    loadFromImage(data: RGBAImage, encoding: ?string) {
```

Comment:
this typing can also be `"mapbox | "terrarium"` – sorry I missed it earlier!

Reply:
Done